### PR TITLE
Integrate new mutations tab

### PR DESCRIPTION
**Pull request by adamabeshouse**
# Checks
- [ ] Runs on Heroku.
- [ ] Follows [7 rules of great commit messages](http://chris.beams.io/posts/git-commit/). For most PRs a single commit should suffice, in some cases multiple topical commits can be useful. During review it is ok to see tiny commits (e.g. Fix reviewer comments), but right before the code gets merged to master or rc branch, any such commits should be squashed since they are useless to the other developers. Definitely avoid [merge commits, use rebase instead.](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [ ] Follows the [Google Style Guide](https://github.com/google/styleguide).
- [ ] Make sure your commit messages end with a Signed-off-by string (this line
  can be automatically added by git if you run the `git-commit` command with
  the `-s` option)
- [ ] If this is a feature, the PR is to rc. If this is a bug fix, the PR is to
  hotfix.

### CODE REVIEW

**Comment by adamabeshouse** [comment]
@alisman how hard would this be to add to MSKTabs? 
image

**Comment by onursumer** [comment]
http://maslianok.github.io/react-responsive-tabs/ 
@adamabeshouse @alisman how about something like this?

File: portal/src/main/webapp/WEB-INF/jsp/visualize.jsp [hunk 1]
```diff
@@ -45,6 +45,14 @@ window.maxTreeDepth = '<%=GlobalProperties.getMaxTreeDepth()%>';
 window.skinExampleStudyQueries = '<%=GlobalProperties.getExampleStudyQueries().replace("\n","\\n")%>'.split("\n");
 
 window.priorityStudies = {};
+
+// global variables required for MutationMapper annotation column
+window.showCivic = <%=GlobalProperties.showCivic()%>;
```
Comment (onursumer) [inline portal/src/main/webapp/WEB-INF/jsp/visualize.jsp]: I think, for now, we should move the global variables related to mutation mapper into a separate jsp file. Then we can include the same jsp file in the cross cancer page as well.

Comment (adamabeshouse) [inline portal/src/main/webapp/WEB-INF/jsp/visualize.jsp]: that sounds good to me

Comment (inodb) [inline portal/src/main/webapp/WEB-INF/jsp/visualize.jsp]: I think we should have all props for frontend defined in one file. That's easier then defining global variables per component/page. We use `template.tag` for that. We don't apply the template on all the pages, when we for instance call an exposed render method directly. I guess we can define another template or jsp that just has the imports. That can be included both by `template.tag` and wherever other components are mounted. So  `portal/src/main/webapp/WEB-INF/jsp/visualize.jsp` and `portal/src/main/webapp/WEB-INF/jsp/cross_cancer_results.jsp` should also just use this one file with all the properties.

**Pull request merged**